packages/apps/graph/src/scripts/devnet/config.ts

```diff
@@ -0,0 +1,5 @@
+export default {
+  NETWORK_ID: 'fast-development',
+  CHAIN_ID: '0',
+  PORT: 8080,
+} as const;
```

Comment:
Can we make this a named export?

```suggestion
export const devnetConfig = {
  NETWORK_ID: 'fast-development',
  CHAIN_ID: '0',
  PORT: 8080,
} as const;
```